src/rental.rb

```diff
@@ -0,0 +1,22 @@
+class Rental
+  attr_accessor :date
+  attr_reader :book, :person
```

Comment:
- [Optional] Hi @Diegogagan2587, I think you should have just defined Getters and Setters for book and person instance variable here( `attr_accessor :book, :person ` ) and get rid of lines 13 to 22. This is a useful shortcut :+1: